apps/frontend/src/libs/components/plan-styles/mocks/plan-mocks.ts

```diff
@@ -0,0 +1,36 @@
+import { type Plan, type PlanDay, type Task } from "~/libs/types/types.js";
+
+const ZERO = 0;
+const ONE = 1;
```

Comment:
As far as I remember we already have ZERO and ONE constants so you can just reuse them